evaluator/evaluator/app/routers/api.py

```diff
@@ -10,4 +11,5 @@
 router.include_router(dataset_router)
 router.include_router(task_router)
 router.include_router(leaderboard_router)
+router.include_router(evaluations_router)
 router.include_router(evaluation_router)
```

Comment:
Warum hast du das nicht in den vorhandenen Router `evaluation_router` geschrieben?

Reply:
Benedikt hatte für den leaderboard eine eigene Datei erstellt, deswegen habe ich es für den evaluations Teil auch gemacht. Ich kann es aber natürlich in den evaluation_router verlagern, wie es euch lieber ist :)

Reply:
Glaube der evaluation_router war sogar genau dafür gedacht 😅

Reply:
Upsi. Ja dann kommt es auf jeden Fall da rein 😂